MobileWallet/Screens/Home/HomeViewController.swift

```diff
@@ -556,7 +555,7 @@ extension HomeViewController {
 
         setupGrabber(fpc)
 
-        fpc.contentMode = .fitToBounds
+        fpc.contentMode = .static
```

Comment:
[Avoid abbreviations. The intended meaning for any abbreviation you use should be easily found by a web search.](https://swift.org/documentation/api-design-guidelines/#use-terminology-well)

Reply:
It's sort of agains the use of `tx` all around in Android code too 🤦  but I'm hoping we can count that as an exception because `tx`s are all around.

Reply:
Actually a search for `tx short for` shows `transaction` in acronym results, so I guess `tx` is valid.